docs/architecture/deployment/one-click-upgrade.md

```diff
@@ -0,0 +1,132 @@
+    * This opens a tabular structure, with a list of users and when were those created in the system
+    * On the right top corner of this table, you’ll find the button “Upgrade Version”
+    * Click on “Upgrade Version”
+
+<img src={require('./img/step2.png').default} alt="step2" width="95%" />
+
+
+#### Step 3: Load the “Upgrade Version” pop-up
+
+1. On clicking the “Upgrade Version”, a pop-up will appear on the screen
+2. The pop-up contains a text box “Version”
+
+<img src={require('./img/step3.png').default} alt="step3" width="75%" />
+
+
+#### Step 4: Select from the list of versions releases
+
+1. On clicking upon the inside of the “Version” text box, the user will see the list of releases from the last release the customer is on currently to the latest release deployed by Prophecy.
+2. Refer to the image below
+
+<img src={require('./img/step4.png').default} alt="step4" width="75%" />
+
+
+#### Step 5: Select the version user wants to upgrade to
+
+1. Select the version user wants to platform to be upgraded to
+2. Since prophecy can connect to URL
+3. Since Prophecy can connect to its Helm repository, we’ll get list of all versions that are available in the drop-down
+
+<img src={require('./img/step5.png').default} alt="step5" width="75%" />
+
+
+#### Step 6: Click on Upgrade
+
+1. Since it is a non-dark cluster, the selected package is retrieved from Prophecy’s Helm repository
+2. Click on Upgrade
+
+<img src={require('./img/step6.png').default} alt="step6" width="75%" />
+
+
+#### Refer [Appendix B for Disable Rollback](#appendix---a--one-click-downgrade)
+
+On clicking Upgrade, the customer will get to see the following screen
+
+<img src={require('./img/step7.png').default} alt="step7" width="75%" />
+
+
+### One-Click Upgrade on Dark Clusters (not connected to internet)
+
+* These clusters do not have access to the internet, hence users can’t see what all versions are released.
+* In this case, the Version list would be empty
+    1. hence, the user can manually put the release number in the “Version” Text box and can trigger the “Upgrade”
+* We package the helm charts with one Prophecy service, named Athena, not to be confused with AWS Athena.
+    1. Helm charts are basically the next package to which to be upgraded.
+
+    
+### Appendix - A | One-Click Downgrade
+
+---
+
+1. Consider a scenario wherein the version selected for upgrade is not stable or say the latest released version has some build verification test failure, that results into the upgrade failure
+* Now, the user would want to downgrade to the previous version
+    1. If the version to which the user wants to downgrade is among the latest version, they can select from the Version drop-down options.
+    2. Else, for the older version they need to manually enter the version
+
+Note:
+
+1. We do not recommend and expect the user to downgrade without asking the Prophecy Cloud team, as things might not be backward compatible and the environment may become unstable.
```

Comment:
Are you sure you want to let the public know which team they need to ask for support? Do you want to say "without asking the Prophecy team" @JKChhabra ?